src/collection_functions.php

```diff
@@ -1390,7 +1390,7 @@ function ($collection) {
  * Transpose each item in a collection, interchanging the row and column indexes.
  * Can only transpose collections of collections. Otherwise an InvalidArgument is raised.
  *
- * @param Collection $collection
+ * @param iterable<Collection> $collection
```

Comment:
`Collection[]` should be fine. The docs only talk about arrays, not iterable with the specific typehint.